tools/releases/helm.sh

```diff
@@ -75,6 +75,13 @@ function package {
 }
 
 function release {
+  if [ "${dev}" == "true" ]; then
```

Comment:
Hmm shouldn't we just not run this if it's not supposed to run?

Reply:
I don't mind (in practice it doesn't run as it's gated in the action).

Reply:
then IMO it's just noise